presto-main/src/test/java/com/facebook/presto/operator/TestHashAggregationOperator.java

```diff
@@ -816,6 +829,131 @@ private void testMemoryTracking(boolean useSystemMemory)
         assertEquals(driverContext.getMemoryUsage(), 0);
     }
 
+    @Test
```

Comment:
Can you think of more tests for this ? 

For example, anything to verify that the "vacous group-by" results output by the partial agg into the output buffer are consumable by the final agg for different data types ?

Reply:
We do test several data types in the unit tests. For comprehensiveness, I think this is better suited for verifier testing, which will test wide range of data types